stable/eventstore/templates/eventstore-statefulset.yaml

```diff
@@ -13,6 +13,7 @@ spec:
       app.kubernetes.io/name: {{ include "eventstore.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name }}
   replicas: {{ .Values.clusterSize }}
+  serviceName: {{ include "eventstore.fullname" . }}
```

Comment:
@chriskolenko hi, I wonder if eventstore._serviceName_ could be a better name for this new setting?